src/task.py

```diff
@@ -24,59 +25,116 @@ class SCLStructruralHabitat(SCLTask):
             "ee_path": "projects/SCL/v1/Panthera_tigris/zones",
             "static": True,
         },
-        "lc_elev_reclass_esa": {
-            "ee_type": SCLTask.FEATURECOLLECTION,
-            "ee_path": "projects/SCL/v1/Panthera_tigris/landcover_reclass_lookup/lc_elev_reclass_esa",
+        "forest_height": {
+            "ee_type": SCLTask.IMAGECOLLECTION,
+            "ee_path": "projects/SCL/v1/Panthera_tigris/source/Hansen_Forest_Height",
+            "maxage": 5,
+        },
+        "watermask": {
```

Comment:
`watermask` is now defined in the base `SCLTask`. Make sure you update your build.